src/components/Charts.tsx

```diff
@@ -94,33 +125,31 @@ function getBarChartData(
    */
 
   const histogram: Histogram = { bars: [] };
-
   const jsonData = JSON.parse(histogramData.get(site));
 
   const metricData = jsonData ? jsonData[metric]?.data : null;
 
   metricData?.forEach((bar: any) => {
-    if (bar.length <= 3000) {
-      histogram.bars.push({ x: bar.length, y: bar.count, fill: color });
-    }
+    // if (bar.length <= 3000) {
+    histogram.bars.push({ x: String(bar.length), y: bar.count, fill: color });
+    // }
   });
   return histogram;
 }
 
 /**
  * getCandleChartData parse and get the data for the correct metric and sites.
  *
- * @param metric a string with the name of the metric to show in the candlechart.
+ * @param metrics a string array with the name of the metrics to show in the candlechart.
  * example 'getPatient'
- * @param sites a string list containing 1-n sites that will be shown in the candlechart.
- * example ['s1','s2','s3','s4']
+ * @param site a string with the name of the site to get data from
  * @param boxDiagramData data from backend.
  * @param siteProps map ecah siteKey to a color
  * @returns a data structure in correct format to paint a candleChart.
  */
-function getCandleChartData(
-  metric: string,
-  sites: Array<string>,
+function getCandleChartData( // rewrite this function
```

Comment:
IMO either rewrite it now or create a task to rewrite it. Otherwise it will likely not happen